src/content/docs/style-guide/writing-guidelines/five-questions-help-write-docs.mdx

```diff
@@ -281,7 +281,9 @@ Each of the five questions includes sub-questions to help guide your thinking.
     id="why"
     title="5. Is the language approachable, expert, and visionary?"
   >
-    Think about **why** the information matters and why readers will trust and use it. For more information, see the [Marketing brand guidelines PDF (34.7M)](https://drive.google.com/file/d/17qsjDFuDv-lcjcbLCeIko5xZwxhw5CiQ/view). (This is an internal New Relic link.)
+    Think about **why** the information matters and why readers will trust and use it. For tips to make it easier for users to read, understand, and use documentation, go to [plainlanguage.gov](https://www.plainlanguage.gov/).
+    
+    Also if you are a New Relic employee, refer to the [Marketing brand guidelines PDF (34.7M)](https://drive.google.com/file/d/17qsjDFuDv-lcjcbLCeIko5xZwxhw5CiQ/view). (This is an internal New Relic link.)
```

Comment:
We've been encouraged to remove all of our links to internal New Relic stuff.

Reply:
Good catch! Fixed with next commit.